Ship_Game/GameScreens/FleetDesign/FleetDesignScreen_Draw.cs

```diff
@@ -355,8 +356,7 @@ void DrawSelectedData(SpriteBatch batch, DrawTimes elapsed)
                 RequisitionForces.Visible = true;
                 SaveDesign.Visible = true;
                 LoadDesign.Visible = true;
-                if (f.Ships.Count > 0 )
-                    AutoArrange.Visible = true;
+                AutoArrange.Visible = f.DataNodes.Where(n => n.Ship == null && n.Goal == null).Count() == 0;
```

Comment:
Seems This is not related to this PR

Reply:
yes it works if all ships are built, so no templates just complete ships